navbar.ts

```diff
@@ -1,4 +1,4 @@
-const mainNav = {
+const navbar = {
```

Comment:
https://github.com/nl-design-system/documentatie/blob/main/navConfig.ts#L3

er is een type hier voor

Reply:
Had je https://github.com/nl-design-system/gebruikersonderzoeken/pull/119/files#diff-cc8abb6104e21d495dc8f64639c7b03419226d920d1c545df51be9b0b73b2784R95 gezien?